src/components/surveys/survey-design/question-edit/rhs-subcontrols/YearRadioGroup.tsx

```diff
@@ -0,0 +1,112 @@
+import {SimpleTextInput, SimpleTextLabel} from '@components/widgets/StyledComponents'
+import {Box, FormControl, FormControlLabel, Radio, RadioGroup} from '@mui/material'
+import {theme} from '@style/theme'
+import React from 'react'
+
+export const DEFAULT_MIN_YEAR = 1900
+export const DEFAULT_MAX_YEAR = new Date().getFullYear()
+
+export type YearFormatType = 'ANY' | 'CURRENT' | 'SET'
+
+const capitalizeFirstLetter = (val: string) => {
+  return `${val.charAt(0).toUpperCase()}${val.substring(1)}`
+}
+
+const yearFormatToAllowValueMap: Record<YearFormatType, boolean | undefined> = {
+  SET: undefined,
+  ANY: true,
+  CURRENT: false,
+}
```

Comment:
Mapping between the allowed value (e.g. `allowPast`) and the selected year format.